packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -373,6 +504,19 @@ export class LitDevTutorial extends LitElement {
       CODE_LANGUAGE_CHANGE,
       this._onCodeLanguagePreferenceChanged
     );
+
+    const previewElement = this._preview!;
+    // must be lowercase or else always undefined
+    const tagname = previewElement.tagName.toLowerCase();
```

Comment:
Can use `localName` instead of `tagName.toLowerCase()`. It's lowercase to begin with.